ocaml/parsing/jane_syntax_parsing.ml

```diff
@@ -786,6 +787,10 @@ module type AST = sig
     of_ast_internal:(Feature.t -> ast -> 'a option) -> ast -> 'a option
 end
 
+let needs_extra_checks = function
```

Comment:
This wants a comment. What are extra checks, and why is it not uniform for all extensions? (I know the answer, but the future reader won't.)